src/test_utils/katana/genesis.rs

```diff
@@ -230,7 +228,9 @@ impl KatanaGenesisBuilder<Loaded> {
             (storage_addr(KAKAROT_NATIVE_TOKEN_ADDRESS)?, *DEFAULT_FEE_TOKEN_ADDRESS),
             (storage_addr(KAKAROT_ACCOUNT_CONTRACT_CLASS_HASH)?, account_contract_class_hash),
             (storage_addr(KAKAROT_UNINITIALIZED_ACCOUNT_CLASS_HASH)?, uninitialized_account_class_hash),
-            (storage_addr(KAKAROT_PRECOMPILES_CLASS_HASH)?, precompiles_class_hash),
+            //TODO: rename the precompiles class hash to cario1_helpers_class_hash in kakarot
+            //https://github.com/kkrt-labs/kakarot/blob/d965e918ffe90671272839d3a8e5035e7bf259e7/src/kakarot/kakarot.cairo#L24-L32
```

Comment:
can you open an issue on the other repo:)?

and reference the issue in this line of code instead?
nice find!

Reply:
ok